conans/test/build_helpers/cmake_test.py

```diff
@@ -1003,6 +1003,32 @@ def test_cmake_system_version_android(self):
             cmake = CMake(conan_file)
             self.assertEquals(cmake.definitions["CMAKE_SYSTEM_VERSION"], "32")
 
+    def install_definitions_test(self):
+        conanfile = ConanFileMock()
+        conanfile.package_folder = None
+        conanfile.settings = Settings.loads(default_settings_yml)
+        install_defintions = {"CMAKE_INSTALL_PREFIX": conanfile.package_folder,
+                              "CMAKE_INSTALL_BINDIR": "bin",
+                              "CMAKE_INSTALL_SBINDIR": "bin",
+                              "CMAKE_INSTALL_SBINDIR": "bin",
+                              "CMAKE_INSTALL_LIBEXECDIR": "bin",
+                              "CMAKE_INSTALL_LIBDIR": "lib",
+                              "CMAKE_INSTALL_INCLUDEDIR": "include",
+                              "CMAKE_INSTALL_OLDINCLUDEDIR": "include",
+                              "CMAKE_INSTALL_DATAROOTDIR": "res"}
```

Comment:
any reason to use "res" instead of "share" for datarootdir? I would expect it will require lots of programs to change location then

Reply:
"res" is the default folder name dor resources and data files in `self.cpp_info` https://docs.conan.io/en/latest/reference/conanfile/methods.html#cpp-info

Reply:
in this case, may be it's better to add "shared" as a secondary default in addition to the existing "res"? "res" sounds to be too foreign for autotools-based installations.

Reply:
Normally CMake package-config files are installed to `/share`:
`
install(
  FILES 
  ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake 
  ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake
  DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/${PROJECT_NAME}
)
`

With this change, they are installed to `/res` which is not a default search path of the CMake `find_package` command: https://cmake.org/cmake/help/v3.12/command/find_package.html?highlight=find_package

Hence, packages are not found anymore if included via `conan_basic_setup()`!!!

Reply:
You were right, I am proposing a fix for this in #3705

Reply:
Great - many thanks!